.github/workflows/ci.yml

```diff
@@ -49,6 +49,12 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v3
 
+    - name: Login to Docker Hub
+      uses: docker/login-action@0d4c9c5ea7693da7b068278f7b52bda2a190a446
```

Comment:
Pinned to https://github.com/docker/login-action/commit/0d4c9c5ea7693da7b068278f7b52bda2a190a446.

Reply:
If you put a version you pinned to after the pin itself with `# $version` like https://github.com/brave/pull-merge/blob/main/.github/workflows/loop-anthropic.yml#L15 you will get automatic updates too

Reply:
Added, thanks!